README.md

```diff
@@ -84,7 +84,18 @@ namespace TwitchLib.EventSub.Websockets.Test
 
             if (!e.IsRequestedReconnect)
             {
-                // subscribe to topics
+                /*
+                Subscribe to topics via the TwitchApi.Helix.EventSub object.
+
+                var conditions = new Dictionary<string, string>()
+                {
+                    { "broadcaster_user_id", someUserId }
+                };
+                var subscriptionResponse = await TwitchApi.Helix.EventSub.CreateEventSubSubscriptionAsync("channel.subscribe", "1", conditions,
```

Comment:
It's a little pedantic, but I'd use channel follow as the example topic, because that's the event the example has a handler for

Reply:
Thats fine, I just picked a random simple one from the twitch sub list, feel free to edit it to whatever you want. I just want to give the reader the knowledge that you will need to call that method with whatever sub stuff you want.

Reply:
I have pushed another update which uses the follow and correctly shows the version 2 of it.